automation_tools/satellite6/hammer.py

```diff
@@ -386,3 +386,21 @@ def attach_subscription_to_host_from_satellite(
         organization_id, product_name)
     return hammer('host subscription attach --subscription-id {0} '
                   '--host {1}'.format(subscription_id, hostname))
+
+
+def hammer_determine_cv_and_env_from_ak(ak_name, organization_id):
```

Comment:
I see, that is not clear by reading the variable name being returned. But I saw that the dict contains both information.

Since that same dict have other information I continue thinking that a the function could have a better name. But this is not a big deal and I won't block the PR to be merged just because this.